desktop/ui/src/test/resources/documentCreatorBenchmarkTest.html

```diff
@@ -0,0 +1,265 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<link rel="stylesheet" type="text/css" href=documentationReference.css />
```

Comment:
This will not be resolveable except when running from the sources. Be aware that as soon as the code is running in production we will only have the packaged JAR file left and non of the resources that you put in the source tree.